chapters/en/module3.md

```diff
@@ -134,7 +134,7 @@ When you are cloning a repository you are downloading a <code>.git</code> hidden
 </exercise>
 
 <exercise id='5' title="The staging area"  type='slides, video'>
-<slides source='module3/module3_03' shot='8' start='3:42' end='4:35'> </slides>
+<slides source='module3/module3_03' shot='8' start='14:35' end='19:59'> </slides>
```

Comment:
This video starts with you saying "Let's apply what we learned" from the following slide deck, maybe shift over 1-2 seconds?

This video also stops in the middle of a sentence